lib/serializables/log-event.js

```diff
@@ -51,9 +51,6 @@ function LogEventFactory (
             host: {
                 type: 'string'
             },
-            module: {
```

Comment:
I think the `module` does provides useful information, as it quickly tells you which module generates the message and so you can quickly go to the source code.

Suppose if there is a log message "running xxxx", if the module is removed from logging entry, then you need to search the whole code repositories to find out which module generates this message, if there are multiple modules generate same message, then you will be lost and you need additional information to locate which module it is.